pages/task/content/status.js

```diff
@@ -200,12 +209,14 @@ module.exports = {
     action: {
       application: 'Edit and resubmit the application',
       amendment: 'Edit and resubmit the amendment',
-      transfer: 'Edit and resubmit the transfer request'
+      transfer: 'Edit and resubmit the transfer request',
+      review: 'Edit and resubmit the review request'
```

Comment:
Is that a thing or is this content just here for completeness?

Reply:
not a thing, will remove